tests/e2e/actions.go

```diff
@@ -936,14 +1116,20 @@ func (tr TestRun) addIbcChannelHermes(
 	action addIbcChannelAction,
 	verbose bool,
 ) {
+	//#nosec G204 -- Bypass linter warning for spawning subprocess with cmd arguments.
+	chanVersion := action.version
+	if chanVersion == "" {
+		chanVersion = tr.containerConfig.ccvVersion
+	}
+
 	//#nosec G204 -- Bypass linter warning for spawning subprocess with cmd arguments.
 	cmd := exec.Command("docker", "exec", tr.containerConfig.instanceName, "hermes",
 		"create", "channel",
 		"--a-chain", string(tr.chainConfigs[action.chainA].chainId),
 		"--a-connection", "connection-"+fmt.Sprint(action.connectionA),
 		"--a-port", action.portA,
 		"--b-port", action.portB,
-		"--channel-version", tr.containerConfig.ccvVersion,
+		"--channel-version", chanVersion,
```

Comment:
making this change to accomodate the fact that the func is used to create IBC `transfer` channels which have a versioning schema (`ICS20-<chan-number>`)